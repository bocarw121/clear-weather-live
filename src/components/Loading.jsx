import React from "react";
import { StyleSheet, View, ActivityIndicator } from "react-native";

import colors from "../constants/colors";

const Loading = ({ size }) => {
  return (
    <View style={[styles.container, styles.horizontal]}>
      <ActivityIndicator color={colors.white} size={size} />
    </View>
  );
};

export default Loading;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    backgroundColor: colors.linearGradient[0],
  },
  horizontal: {
    flexDirection: "row",
    justifyContent: "space-around",
    padding: 10,
  },
});
