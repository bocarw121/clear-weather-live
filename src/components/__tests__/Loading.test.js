import React from "react";

import { render } from "@testing-library/react-native";

import Loading from "../Loading";

it("Should render the Loading", () => {
  const screen = render(<Loading />);

  expect(screen).toMatchSnapshot();
});
