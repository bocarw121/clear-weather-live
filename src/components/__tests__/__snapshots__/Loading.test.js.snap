// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Loading 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgb(0, 132, 255)",
        "flex": 1,
        "justifyContent": "center",
      },
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
        "padding": 10,
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#fff"
    hidesWhenStopped={true}
    size="small"
  />
</View>
`;
